doc/src/build_instructions.rst

```diff
@@ -42,6 +42,20 @@ configuration options at hand.
    docs.cppmicroservices.org.
  - **US_BUILD_DOC_MAN** Build the man pages. This is typically only
    enabled on a Unix-like system.
+ - **GTEST_ROOT** Specify the root directory of the Google Test framework
+   installation. These must be specified to build and run unit tests
+   implemented using Google Test. Note that the Google Testing framework must
+   have been built as a shared library.
```

Comment:
Why is it required to be built as a shared library? Shouldn't we be able to support both shared and static, especially when re-using an existing GTest installation?